sections/proof-of-yield/DayBasicData.tsx

```diff
@@ -176,17 +201,18 @@ const DayBasicData = ({ timestamp, sectionOverrideCss }: DayBasicDataProps) => {
                 </tr>
               </thead>
               <tbody>
-                {mockData.map((item) => (
+                {dailyStat.rebase_events.map((item) => (
```

Comment:
Should get use to `?` just to safeguard against any client errors throwing on `undefined`